### ISSUE TITLE
improve content accessibility by making [here] links more descriptive

### ISSUE THREAD

**Issue by serahkiburu** [carpentries/docs.carpentries.org#702]
As @zkamvar rightly pointed out in https://github.com/carpentries/carpentries.org/pull/867 a few months ago, having a link point to "here" is not accessible for users who use screen-readers to scan for links: https://webaccess.berkeley.edu/ask-pecan/click-here

We invite you to help us make our content more accessible by [creating Pull Requests](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request) in this repository with suggestions for more descriptive text to replace [here] links across our resource. 

(not required but nice to have)Where possible, you can also [mention this issue in your Pull Request](https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue) to enable us to keep track of accessibility improvements made on this specific issue.

Thank you for your contributions.

**Comment by maneesha** [comment]
Assigning this to @brynnelliott - not sure what your workflow or workload is like, so prioritize as you see fit

**Comment by maneesha** [comment]
Checking in @brynnelliott 
Is this something you'd like to keep open to work on in the current handbook, or focus on the new one?  Should we be ensuring that we never use contractions?

**Comment by brynnelliott** [comment]
@maneesha I think we focus on this in the new handbooks. I think I have taken care of most of the 'click here' 

As for contractions, I think they are ok. After much research, they can actually make things easier to understand. Screen readers are able to read them. Other than being more informal, from an accessibility/ableist perspective, I don't think we need to worry about them.
